ddtrace/contrib/redis/__init__.py

```diff
@@ -30,7 +30,7 @@
 Instance Configuration
 ~~~~~~~~~~~~~~~~~~~~~~
 
-To configure particular redis instances use the :ref:`Pin<Pin>` API::
+To configure particular redis instances use the :class:`Pin <ddtrace.Pin>` API::
```

Comment:
```suggestion
To configure particular redis instances use the :class:`Pin<ddtrace.Pin>` API::
```

?

Reply:
Convention is to add a space between the text and the link. https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref